packages/office-ui-fabric-react/src/components/Spinner/Spinner.styles.tsx

```diff
@@ -48,7 +48,7 @@ export const getStyles = (props: ISpinnerStyleProps): ISpinnerStyles => {
       {
         boxSizing: 'border-box',
         borderRadius: '50%',
-        border: '1.5px solid ' + palette.themeLight,
+        border: '1.5px solid ' + semanticColors.inputPlaceholderBackgroundChecked,
         borderTopColor: palette.themePrimary,
```

Comment:
A really weird choice of semantic slot. Why having one for border and none for border-top? Even though the top-border is of a darker shade (themePrimary) than the rest of the border, we still use the `checked` slot on the lighter part of the spinner. I would think it should be the opposite way 🤔